pytorch_lightning/trainer/trainer.py

```diff
@@ -1223,7 +1223,6 @@ def _log_hyperparams(self) -> None:
             self.logger.save()
 
     def _post_dispatch(self):
-        self.accelerator.post_dispatch(self)
         # these `teardown` calls are here instead of in `_call_teardown_hook` since they are internal teardowns
         # which need to happen before.
         self.accelerator.teardown()
```

Comment:
What used to be post_dispatch is now handled here with the call to teardown(). As part of the final steps in #10416, this will then become `self.strategy.teardown()` eventually.